ui/src/components/AuthProvider.tsx

```
@@ -163,21 +165,19 @@ class Auth {
     try {
```

Should we throw an error for the warning on L120 as well, when fetching `/auth/login_info` fails? (Apologies, GitHub won't let me comment on that line of the diff.)

---

This is a warning... and in the prod code will result in a warning and a return from login. 
Throwing will result in an error - which I think is more correct, but since you wrote the orig logic, what do you think?